### PR TITLE
Return variable in get_url is 'status_code', not 'status'

### LINKED ISSUE #43365: get_url docs specifies return var 'status' which actually is 'status_code'
##### SUMMARY
The docs for get_url describe as one of the return variables:

```status; always; the HTTP status code from the request```

This will never be returned; the source code doesn't return it. What it should specify is:

```status_code; always; the HTTP status code from the request```

##### ISSUE TYPE
 - Documentation Report

##### COMPONENT NAME
get_url

##### ANSIBLE VERSION
devel

##### OS / ENVIRONMENT
N/A
##### STEPS TO REPRODUCE
Seen in documentation

### PR DESCRIPTION
**Pull request by skylerbunny**
Subject is self explanatory.
Resolves #43365.

+label: docsite_pr

##### ISSUE TYPE

 - Feature Pull Request
 - New Module Pull Request
 - Bugfix Pull Request
 - Docs Pull Request

##### ANSIBLE VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/modules/net_tools/basics/get_url.py [hunk 1]
```diff
@@ -269,7 +269,7 @@
     returned: success
     type: string
     sample: file
-status:
+status_code:
```
Comment (s-hertel) [inline lib/ansible/modules/net_tools/basics/get_url.py:272]: LGTM. It doesn't look like `state` gets returned either.

Comment (skylerbunny) [reply lib/ansible/modules/net_tools/basics/get_url.py:272]: IIRC, I saw it return <code>state: "absent"</code> once when I got a "Connection reset by peer" error. I guess that is to say I'm not really sure what its relationship is to when it should actually be returned.

**Comment by skylerbunny** [comment]
Ping, simple one line documentation fix.

**Comment by ptux** [comment]
thanks to the PR, I've noticed the problem that there is a big mismatch between code and docs.
I open a new issue #44238.

**Pull request merged**